models/demos/wormhole/squeezebert/tests/test_performance.py

```diff
@@ -0,0 +1,151 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+
+import time
+import ttnn
+import torch
+import pytest
+import transformers
+from loguru import logger
+from models.perf.perf_utils import prep_perf_report
+from ttnn.model_preprocessing import preprocess_model_parameters
+from models.demos.wormhole.squeezebert.tt import ttnn_functional_squeezebert
+from models.experimental.functional_common.attention_mask_functions import get_extended_attention_mask
+from models.utility_functions import (
+    enable_persistent_kernel_cache,
+    disable_persistent_kernel_cache,
+    is_wormhole_b0,
+    skip_for_grayskull,
+)
+
+
+def synchronize_devices(device):
+    devices = device.get_devices()
+    for device in devices:
+        ttnn.synchronize_device(device)
+
+
+def get_expected_times(squeezebert):
+    return {ttnn_functional_squeezebert: (29.29, 15.5)}[squeezebert]
```

Comment:
what's the current run time? How close is it to the expected times?

Reply:
The current runtimes in the test file have been updated based on the average times observed during CI runs.
We’re uncertain about the target numbers. 

cc: @boris-drazic, @mbahnasTT.

Reply:
You want the expected times to be close to the current time, but not so close that a small variation will cause test to fail. Maybe 10-20% margin is good.